### PR TITLE
Haf 50 integrate add keys page with sc

### PR DESCRIPTION
**Pull request by KacperKoza343**
## Related issues
[Keywords for linking issues](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)

### CODE REVIEW

File: packages/apps/human-app/frontend/src/smart-contracts/EthKVStore/get-all-keys.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Contract } from 'ethers';
+import EthKVStore from '@/smart-contracts/abi/EthKVStore.json';
+import type { ContractCallArguments } from '@/smart-contracts/types';
+import { EthKVStoreKeys } from '@/smart-contracts/EthKVStore/config';
+
+export async function get({
```
Comment (MicDebBlocky) [inline packages/apps/human-app/frontend/src/smart-contracts/EthKVStore/get-all-keys.ts]: change name for get-eth-all-keys. Change also file name

File: packages/apps/human-app/frontend/src/smart-contracts/EthKVStore/set-bulk.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Contract } from 'ethers';
+import EthKVStore from '@/smart-contracts/abi/EthKVStore.json';
+import type { ContractCallArguments } from '@/smart-contracts/types';
+import type {
+  SetBulkKeys,
+  SetBulkValues,
+} from '@/smart-contracts/EthKVStore/config';
+
+export async function setBulk({
```
Comment (MicDebBlocky) [inline packages/apps/human-app/frontend/src/smart-contracts/EthKVStore/set-bulk.ts]: setEthBulk

File: packages/apps/human-app/frontend/src/components/data-entry/multi-select.tsx [hunk 3]
```diff
@@ -82,6 +86,13 @@ export function MultiSelect({
     field: FieldType
   ) => {
     const value = event.target.value;
+    if (singleValue) {
```
Comment (MicDebBlocky) [inline packages/apps/human-app/frontend/src/components/data-entry/multi-select.tsx]: to remove

File: packages/apps/human-app/frontend/src/pages/operator/sign-up/add-keys/existing-keys.tsx [hunk 4]
```diff
@@ -5,6 +5,27 @@ import { Chips } from '@/components/ui/chips';
 import { Button } from '@/components/ui/button';
 import { colorPalette } from '@/styles/color-palette';
 import type { UseFormResult } from '@/pages/operator/sign-up/add-keys/add-keys.page';
+import { EthKVStoreKeys } from '@/smart-contracts/EthKVStore/config';
+
+function EmptyPlaceholder() {
```
Comment (MicDebBlocky) [inline packages/apps/human-app/frontend/src/pages/operator/sign-up/add-keys/existing-keys.tsx]: move to global components folder

File: packages/apps/human-app/frontend/src/pages/operator/sign-up/add-keys/existing-keys.tsx [hunk 5]
```diff
@@ -14,35 +35,43 @@ export function ExistingKeys({
   openEditMode: () => void;
 }) {
   const { getValues } = useFormResult;
```
Comment (MicDebBlocky) [inline packages/apps/human-app/frontend/src/pages/operator/sign-up/add-keys/existing-keys.tsx]: if useFormResult is not a hook, should has name like getFormValues

File: packages/apps/human-app/frontend/src/components/data-entry/select.tsx [hunk 6]
```diff
@@ -17,13 +18,15 @@ interface SelectComponentProps extends Omit<SelectProps, 'name' | 'error'> {
   name: string;
   label?: string;
   ariaLabelledby?: string;
+  chip?: boolean;
```
Comment (MicDeb) [inline packages/apps/human-app/frontend/src/components/data-entry/select.tsx]: Change name to more descriptive, explaining

**Pull request merged**